lld/wasm/Writer.cpp

```diff
@@ -1366,6 +1366,14 @@ void Writer::createInitMemoryFunction() {
           writeUleb128(os, s->index, "segment index immediate");
           writeU8(os, 0, "memory index immediate");
         }
+
+        // After initializing the TLS segment, we also need to apply TLS
```

Comment:
How about "After initializing the TLS segment and setting `__tls_base` we can call `__wasm_apply_global_tls_relocs`"